packages/back-end/src/models/SegmentModel.ts

```diff
@@ -56,5 +83,11 @@ export async function updateSegment(
   organization: string,
   updates: Partial<SegmentInterface>
 ) {
+  // If using config.yml & the org doesn't have the env variable STORE_SEGMENTS_IN_MONGO,
+  // immediately return the list from there
+  if (usingFileConfigForSegments()) {
```

Comment:
minor: should we also have something in the UI that shows why we can't add/remove segments when using a config file?

Reply:
@bryce-fitzsimons Good rec - I just created an issue for that [here](https://github.com/growthbook/growthbook/issues/1051) as a follow-up.